wikimedia/wikipedia-ios
# Remove emdash workaround

**Phabricator:** https://phabricator.wikimedia.org/T361868

### Notes
I clearly did this for a reason in the past, but I can no longer reproduce the find and replace bug mentioned in the comment. Removing the workaround.

### Test Steps
1. Open native editor
2. Long press on hyphen in keyboard - press emdash (longest dash) character.
3. Confirm character inserts.
